.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+## Is your feature request related to a problem? Please describe the impact that the lack of the feature requested is creating.
+A clear and concise description of what the problem or need is. E.g. I need to...
```

Comment:
Please wrap the answers that are be removed in this file in []?
`[A clear and concise description of what the problem or need is. E.g. I need to...]`